pages/person/_itemId/index.vue

```diff
@@ -5,12 +5,13 @@
         <v-row>
           <v-col cols="2">
             <v-img
-              ref="personImg"
-              class="person-image elevation-2 ml-2"
+              v-if="item"
               cover
               aspect-ratio="1"
-              :src="getImageUrl(item.Id)"
-            />
+              class="person-image elevation-2 ml-2"
+            >
+              <blurhash-image :item="item" />
+            </v-img>
```

Comment:
Why is `<blurhash-image />` inside a `<v-img>` here?

Reply:
Same as above